lib/outbound/index.js

```diff
@@ -42,25 +42,27 @@ Outbound.prototype.loaded = function(){
 };
 
 Outbound.prototype.identify = function(identify){
-  var firstLevelAttributes = ['user_id', 'first_name', 'last_name', 'email', 'phone_number', 'apns', 'gcm'];
-  var allAttributes = identify.traits();
-  var userAttributes = {
-    attributes: {}
+  var userAttributes = function(identify){
```

Comment:
This function gets reinstantiated every time `identify()`gets called and doesn't really add clarity—just inline the code